src/collections/Pages/components/DuplicatePageFor/index.tsx

```diff
@@ -1,18 +1,23 @@
 import { byGlobalRole } from '@/access/byGlobalRole'
-import { User } from '@/payload-types'
 import { roleAssignmentsForUser } from '@/utilities/rbac/roleAssignmentsForUser'
 import { ruleMatches } from '@/utilities/rbac/ruleMatches'
 import { EditMenuItemsServerProps, PayloadRequest } from 'payload'
 import { DuplicatePageForDrawer } from './DuplicatePageForDrawer'
 
 export const DuplicatePageFor = ({ user, payload }: EditMenuItemsServerProps) => {
+  if (!user) {
+    return null
+  }
+
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
eg here - why did we need the `as`? if it's to omit otherwise required fields, how do we know that all downstream users don't need any of those? if they don't, wouldn't the real fix be to make the methods we pass this into require a smaller subset of this type?

Reply:
See this [playground](https://www.typescriptlang.org/play/?#code/JYOwLgpgTgZghgYwgAgM4FcAO1UTMgbwChlTkZgIAbAEwEYAuNMKUAcwG4SyLqaAmJgCMA9iKoQ4ILgF8iRUJFiIUGIbnzEy5SrUHJR4ydKJyiMdCARhgIkMgR2MAWwgAKVEzUaAlIW6kjiCoRgB0VCJsHqG8ej6m8kGo+GAAFuxeWDh4yAC8hLH0TABEAO6pInDFADQ6fPrwVLgyXEnorm5p7D5AA)